plugin/k8s_extenral: Supports fallthrough option (#5959)
miekg/coredns: plugin/k8s_external/README.md

* if there is a headless service with external IPs set, external IPs will be resolved

> if zone matches and there are not any external IPs, you can fallthrough to next plugin by adding `fallthrough` option.

Reviewer: ```suggestion
If the queried domain does not exist, you can fall through to next plugin by adding the `fallthrough` option.
```

     }
 ~~~

With the fallthrough option, if the query is not found, it will be passed to the next plugin.
~~~
. {
   kubernetes cluster.local
   k8s_external example.org {
     fallthrough

>    }

Reviewer: Should include a next plugin in this example (e.g. _forward_), otherwise the example only accomplishes turning NXDOMAIN responses into SERVFAIL responses.

     }
 ~~~

> With the fallthrough option, if the query is not found, it will be passed to the next plugin that matches the zone.

Reviewer: ```suggestion
With the `fallthrough` option, if the queried domain does not exist, it will be passed to the next plugin that matches the zone.
```

Author: Fixed
